src/overlays/actors/ovl_Bg_Haka_Tomb/z_bg_haka_tomb.c

```diff
@@ -24,15 +24,8 @@ void BgHakaTomb_SetupDoNothing(BgHakaTomb* this);
 void BgHakaTomb_DoNothing(BgHakaTomb* this, PlayState* play);
 
 ActorInit Bg_Haka_Tomb_InitVars = {
-    ACTOR_BG_HAKA_TOMB,
-    ACTORCAT_BG,
-    FLAGS,
-    OBJECT_HAKA_OBJ,
-    sizeof(BgHakaTomb),
-    (ActorFunc)BgHakaTomb_Init,
-    (ActorFunc)BgHakaTomb_Destroy,
-    (ActorFunc)BgHakaTomb_Update,
-    (ActorFunc)BgHakaTomb_Draw,
+    ACTOR_BG_HAKA_TOMB, ACTORCAT_BG,       FLAGS,           OBJECT_HAKA_OBJ, sizeof(BgHakaTomb), BgHakaTomb_Init,
+    BgHakaTomb_Destroy, BgHakaTomb_Update, BgHakaTomb_Draw,
```

Comment:
this is the main reason they have stayed around, for formatting only. not ideal obviously cause its just a hack to make the formatter do what most think looks better, but yea

Reply:
Two other cheating options:

1) make a fake (?) substruct for the actor funcs
```c
ActorInit En_Firefly_InitVars = {
    ACTOR_EN_FIREFLY,
    ACTORCAT_ENEMY,
    FLAGS,
    OBJECT_FIREFLY,
    sizeof(EnFirefly),
    {
        EnFirefly_Init,
        EnFirefly_Destroy,
        EnFirefly_Update,
        EnFirefly_Draw,
    },
};
```

2) put some comment somewhere
```c
ActorInit En_Firefly_InitVars = {
    ACTOR_EN_FIREFLY,
    ACTORCAT_ENEMY,
    FLAGS,
    OBJECT_FIREFLY,
    sizeof(EnFirefly),
    //
    EnFirefly_Init,
    EnFirefly_Destroy,
    EnFirefly_Update,
    EnFirefly_Draw,
};
```

Reply:
Ore we could, y'know, just use the thing that the formatter has for that purpose. Actually I'm surprised no one's threatened to macro these.

Reply:
The problem with macroifying the InitVars is not every actor have the 4 main functions in the InitVars
I guess I could try something and see how it ends up looking

Reply:
Pushed the extra braces change, tell me if you guys prefer it this way

Reply:
I think it looks worse. And for most people positional initialisation is bad enough without substructs, let alone spurious extra ones: you often hear complaints about the collider inits from that PoV.

Reply:
Relevant discussion we had in oot not too long ago: <https://discord.com/channels/688807550715560050/688851317593997489/990310020166090762>
We never really reached a conclusion but the preferred solution seemed to be using `/**/` comments (either empty or with a brief name of each field)

Reply:
Personally I don't really like the substruct approach, and much prefer the comments, even if empty. TBH, I don't really think we even really need to force the formatter to one thing, but can understand why it is wanted.

Reply:
Updated with the `/**/` comments.
I don't think the format of the InitVars matter that much either, but people prefer it this way

Reply:
I don't understand why this is any less messy than explicitly disabling formatting. Indeed, it will make very little sense if you're looking at it without knowledge of its purpose, which implies to me that it's strictly worse.

Reply:
At least for me, this is 1000 times better than explicitly disabling formatting, since it is automated.  If someone makes any changes, they don't have to worry about manually formatting after, instead it will still be handled properly.